src/lib/common/macroSubstitute.cpp

```diff
@@ -209,7 +209,7 @@ bool macroSubstitute(std::string* to, const std::string& from, const Entity& en)
 
   if (from.length() + toAdd - toReduce > outReqMsgMaxSize)
   {
-    LM_W(("Runtime Error (too large final string, after substitution)"));
+    LM_E(("Runtime Error (too large final string, after substitution)"));
```

Comment:
Increased to ERROR due to this is a problem that should be review from an operational point of view (probably enlarging the `-outReqMsgMaxSize` CLI)

NTC (informative)